src/fillTank.test.js

```diff
@@ -1,11 +1,94 @@
 'use strict';
 
+const { fillTank } = require('./fillTank');
+
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  test('should fill the tank only up to its capacity even if'
+    + 'more money is available', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40, fuelRemains: 38,
```

Comment:
"maxTankCapacity: 40," and "fuelRemains: 38," should be on different lines (applies to all tests)

Reply:
fixed